docker-compose.yml

```diff
@@ -83,6 +83,12 @@ services:
       ELASTICSEARCH_ENABLE_REST_TLS: 'false'
       # ELASTICSEARCH_USER is hardcoded to 'elastic' in the container.
       ELASTICSEARCH_PASSWORD: ${ELASTICSEARCH_PASS:-zammad}
+      # Resource settings
```

Comment:
Maybe it is useful to link to the documentation here?
https://github.com/bitnami/containers/tree/main/bitnami/elasticsearch#environment-variables